stac_mjx/op_utils.py

```
@@ -1,11 +1,21 @@
+from jax.lib import xla_bridge
 from mujoco import mjx
 from mujoco.mjx._src import smooth
 import numpy as np
-from stac_mjx import utils
+from stac_mjx import io
+
+
+def enable_xla_flags():
+    """Enables XLA Flags for faster runtime on Nvidia GPUs."""
+    if xla_bridge.get_backend().platform == "gpu":
+        os.environ["XLA_FLAGS"] = (
```

Ruff

Undefined name `os`

(F821)